src/pages/components/tabs/style.mdx

```diff
@@ -33,17 +166,73 @@ Tab label should be set in sentence case, and should not exceed three words.
 
 ## Structure
 
-| Class                 | Property                    | px / rem | Spacing token |
+### Line tabs
```

Comment:
This is the way I would structure this table 

| Element         | Property                    | px / rem | Spacing token |
| --------------- | --------------------------- | -------- | ------------- |
| Tab             | border-bottom               | 2px      | –             |
|                 | margin-left                 | 1px      | —             |
| Label           | padding-left, padding-right | 16 / 1   | `$spacing-05` |
|                 | padding-top, padding-bottom | 8 / 0.5  | `$spacing-03` |
| Tab icon        | padding-right               | 16 / 1   | `$spacing-05` |
|                 | padding-left                | 8 / 0.5  | `$spacing-03` |
|                 | svg (md)                    | 16 x 16  | —             |
|                 | svg (lg)                    | 20 x 20  | —             |
| Scrollable icon | svg                         | 16 x 16  | –             |

Reply:
I added height and weight as per my other comment. I think then the image would read like 
![image](https://user-images.githubusercontent.com/11670886/120722615-a1e14b80-c495-11eb-84d9-709c9d155065.png)

Reply:
I would also maybe hyperlink "auto width" in the table to where you talk about it on the usage tab.